tests/utils.py

```diff
@@ -19,7 +19,15 @@
     from termcolor import colored as colored
 except ImportError:
 
-    def colored(text: str, color: str | None = None, on_color: str | None = None, attrs: Iterable[str] | None = None) -> str:
+    def colored(
```

Comment:
I think it's better to just take `**kwargs` or something instead of duplicating all the args, which is very fragile.

Fun fact: Alex complained about this in person to @hugovk over dinner yesterday.

Reply:
Looks like I missed all the fun :(

Reply:
Maybe next time!